src/tests/unit/native_adapter_support_tests.js

```diff
@@ -62,11 +62,11 @@ setup(() => {
   Turbo.registerAdapter(adapter)
 })
 
-test("navigator adapter is native adapter", async () => {
+test("test navigator adapter is native adapter", async () => {
```

Comment:
I think the extraneous `test` in the descriptions were removed after the original changes were added, and so reverting is bringing some of them back. Might we worth removing them again as an additional commit, for consistency?

Reply:
Good point, I've added the commits to the branch.

The branch was gone, so I had to recreate the commits. Definitely one case in which squashing the branch was not very helpful.